### PR TITLE
Feature/13016 ess UI modify exchange set landing page as figma

### PR DESCRIPTION
**Pull request by prasad14878**

### CODE REVIEW

File: src/app/features/exchange-set/ess-add-single-encs/ess-add-single-encs.component.html [hunk 1]
```diff
@@ -0,0 +1,12 @@
+<div class="container">
+<div class="addSingleFileSection">
+    <ukho-textinput class="addSingleFileTextbox" label="Select a single ENC" [placeholder]="placeholder"></ukho-textinput>
```
Comment (crdandoukho) [inline src/app/features/exchange-set/ess-add-single-encs/ess-add-single-encs.component.html:3]: Is there a reason why placeholder is populated by a binding but the label property is not? If no, could you please replace the placeholder binding with the literal string

File: src/app/features/exchange-set/ess-upload-file/ess-upload-file/ess-upload-file.component.scss [hunk 2]
```diff
@@ -0,0 +1,7 @@
+.UploadFileIput{
```
Comment (crdandoukho) [inline src/app/features/exchange-set/ess-upload-file/ess-upload-file/ess-upload-file.component.scss:1]: I notice .UploadFileInput and .UploadButton styles are not nested inside .uploadENCFileSection. Please nest the styles if it's better practice to do so.

**Review by crdandoukho — CHANGES_REQUESTED** [review]
Looks good. 

Just a few comments/questions

File: src/app/features/exchange-set/ess-add-single-encs/ess-add-single-encs.component.html (hunk 1 above)
Comment (prasad14878) [reply src/app/features/exchange-set/ess-add-single-encs/ess-add-single-encs.component.html]: Resolved

File: src/app/features/exchange-set/ess-upload-file/ess-upload-file/ess-upload-file.component.scss (hunk 2 above)
Comment (prasad14878) [reply src/app/features/exchange-set/ess-upload-file/ess-upload-file/ess-upload-file.component.scss]: Resolved

**Review by crdandoukho — APPROVED** [review]
Thanks Prasad for your changes

Looks good

**Review by bhalcha13813 — APPROVED** [review]
LGTM

**Pull request merged**